admin/attributes_controller.php

```diff
@@ -753,25 +754,14 @@ function zen_js_option_values_list($selectedName, $fieldName)
       .red-txt {color: #f00;}
       .black-txt {color: #000;}
     </style>
-    <script src="includes/menu.js"></script>
-    <script src="includes/general.js"></script>
     <script>
       function popupWindow(url) {
         window.open(url, 'popupWindow', 'toolbar=no,location=no,directories=no,status=no,menubar=no,scrollbars=no,resizable=yes,copyhistory=no,width=600,height=460,screenX=150,screenY=150,top=150,left=150')
       }
     </script>
-    <script>
-      function init() {
-        cssjsmenu('navbar');
-        if (document.getElementById) {
-          var kill = document.getElementById('hoverJS');
-          kill.disabled = true;
-        }
-      }
-    </script>
   </head>
   <!-- <body onload="init()"> -->
-  <body onload="init()">

```

Comment:
Is this truly safe to remove?

Reply:
Yes, the init function sets the viewport in the head, if it is not already there. 
Since we now use bootstrap, this viewport should always be present, 
```html
<body onload="init()">
```
calls
```javascript
    <script>
      function init() {
          cssjsmenu('navbar');
          if (document.getElementById) {
              var kill = document.getElementById('hoverJS');
              kill.disabled = true;
          }
      }
    </script>
```
cssjsmenu() is set in the file admin/includes/menu.js
menu.js contains
```javascript
/* this is only for backward compatibility - will be removed in future version */
function cssjsmenu() {
  viewport = document.querySelector("meta[name=viewport]");
  if (viewport != undefined) {
    viewport.setAttribute('content', 'width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=1');
  } else {
    var metaTag=document.createElement('meta');
  metaTag.name = "viewport"
    metaTag.content = "width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=1"
    document.getElementsByTagName('head')[0].appendChild(metaTag);
  }
}
function init() {}
```
So if we hard-code the required viewport in the <head>, the above code can safely be removed.
This what I do in all my admin module pages for some time now.